pom.xml

```diff
@@ -46,29 +46,28 @@
 
         <!-- Plugins -->
         <version.microprofile.build-tools>1.2</version.microprofile.build-tools>
-
-        <version.plugin.compiler>3.10.1</version.plugin.compiler>
-        <version.plugin.resources>3.2.0</version.plugin.resources>
-        <version.plugin.jar>3.2.0</version.plugin.jar>
-        <version.plugin.dependency>3.1.2</version.plugin.dependency>
-        <version.plugin.source>3.2.0</version.plugin.source>
-        <version.plugin.javadoc>3.2.0</version.plugin.javadoc>
-
-        <version.plugin.helper>3.3.0</version.plugin.helper>
-        <version.plugin.impsort>1.8.0</version.plugin.impsort>
+        <version.plugin.compiler>3.13.0</version.plugin.compiler>
+        <version.plugin.resources>3.3.1</version.plugin.resources>
+        <version.plugin.jar>3.4.1</version.plugin.jar>
+        <version.plugin.dependency>3.7.0</version.plugin.dependency>
+        <version.plugin.source>3.3.1</version.plugin.source>
+        <version.plugin.javadoc>3.7.0</version.plugin.javadoc>
+        <version.plugin.helper>3.6.0</version.plugin.helper>
+        <version.plugin.impsort>1.10.0</version.plugin.impsort>
```

Comment:
This version of impsort also requires Java 17.

With this line reverted, it does build with Java 11, so the rest should be fine.